package.json

```diff
@@ -18,7 +18,7 @@
     "test": "npm run lint",
     "lint": "eslint **/*.js",
     "lint-ext": "web-ext lint --source-dir src",
-    "start": "web-ext run --source-dir src --firefox=\"C:\\Program Files (x86)\\Firefox Developer Edition\\firefox.exe\" --firefox-profile=dev-edition-default",
+    "start": "web-ext run --source-dir src --firefox=\"C:\\Program Files\\Firefox Developer Edition\\firefox.exe\" --firefox-profile=dev-edition-default",
```

Comment:
Despite being an x64 system, my install is still in `Program Files (x86)` so please revert this.